prow/plugins/lgtm/lgtm.go

```diff
@@ -366,16 +364,18 @@ func handlePullRequest(log *logrus.Entry, gc githubClient, config *plugins.Confi
 					break
 				}
 			}
-			// Get the current tree-hash
-			commit, err := gc.GetSingleCommit(org, repo, *pe.PullRequest.MergeSHA)
-			if err != nil {
-				log.WithError(err).Errorf("Failed to get commit.")
-			}
-			treeHash := commit.Commit.Tree.SHA
-			if treeHash == lastLgtmTreeHash {
-				// Don't remove the label, PR code hasn't changed
-				log.Infof("Keeping LGTM label as the tree-hash remained the same: %s", treeHash)
-				return nil
+			if lastLgtmTreeHash != "" {
+				// Get the current tree-hash
+				commit, err := gc.GetSingleCommit(org, repo, pe.PullRequest.Head.SHA)
+				if err != nil {
+					log.WithField("sha", pe.PullRequest.Head.SHA).WithError(err).Errorf("Failed to get commit.")
```

Comment:
`Error` not `Errorf`.
This applies to most all of the `Errorf` calls in this file.

Reply:
I even wonder why this isn't caught by golint...